contrib/epee/src/readline_buffer.cpp

```diff
@@ -53,6 +53,13 @@ rdln::readline_buffer::readline_buffer()
 void rdln::readline_buffer::start()
 {
   boost::lock_guard<boost::mutex> lock(sync_mutex);
+
+// https://linuxsheet.com/answers/687103/
+// Bracketed paste escape sequences added in by default as of version 8.1
+#if RL_VERSION_MAJOR > 8 || (RL_VERSION_MAJOR == 8 && RL_VERSION_MINOR >= 1)
+  rl_variable_bind("enable-bracketed-paste", "off");
+#endif
```

Comment:
This should ideally be in `install_line_handler` instead of `readline_buffer::start` (and without the comment, as the code is self explanatory).